### PR TITLE
fix: typos in migration docs

### PR DESCRIPTION
**Pull request by mohdashraf010897**
minor typos

### CODE REVIEW

File: the the [hunk 1]
```diff
@@ -692,7 +692,8 @@ You likely want to use the `DocsContainer` component exported by `@storybook/blo
 
 **Overriding theme**:
 
-To override the theme, you can continue to use the `docs.theme` parameter.
+To override 
+me, you can continue to use the `docs.theme` parameter.
```
Comment (jonniebigodes) [inline the the:696]: @mohdashraf010897, I'd appreciate it if you wouldn't mind reverting this to how it was initially. The change you're making here is introducing additional typos.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@mohdashraf010897, thanks for taking the time and effort for this pull request and helping us improve the documentation by fixing the typos. We appreciate it. Just one small item and this should be good to go

**Review by jonniebigodes — COMMENTED** [review]
@mohdashraf010897, thanks for addressing the feedback so promptly. Appreciate it 🙏 ! In the meantime, can you check on your end what's happening it seems that are some conflicts in the file? Let me know, and we'll go from there.

**Review by CasperSocio — APPROVED** [review]

**Comment by jonniebigodes** [comment]
Closing this as it was fixed in a follow up pull request.

**Pull request closed without merging**